packages/editor/src/store/utils/is-template-part-revertable.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { TEMPLATE_ORIGINS } from '../constants';
+
+/**
+ * Check if a template part is revertable to its original theme-provided template file.
+ *
+ * @param {Object} templatePart The template part entity to check.
+ * @return {boolean} Whether the template part is revertable.
+ */
+export default function isTemplatePartRevertable( templatePart ) {
```

Comment:
There's already a function called isTemplateRevertable which I believe is the exact same function (we can reuse it by naming the argument templateOrTemplatePart

Reply:
Updated in [ab9e7b1](https://github.com/WordPress/gutenberg/pull/62951/commits/ab9e7b145e84a743fbabc93f98c40494b5687856).